styles.css

```diff
@@ -71,3 +87,136 @@ a {
 .headline__list_item_icon {
   fill: var(--color-icons);
 }
+
+/* Buttons */
+.btn--primary {
+  background: #ebebff;
+  border-radius: 8px;
+  border-style: none;
+  color: var(--color-blue);
+  font-weight: 500;
+  padding: 4px 12px;
+}
+
+.btn--secondary {
+  background-color: #ffff;
+  border: 1px solid var(--color-primary);
+  border-radius: 8px;
+  color: var(--color-primary);
+  font-weight: 500;
+  padding: 8px 16px;
+}
```

Comment:
- [ ] I think It would be better also to follow the button styling: Active, Hover, and Pressed (see design) as given on the Figma design for the best User Experience 😎

   **Figma button design**
   
   ![image](https://user-images.githubusercontent.com/102757126/219433038-ddffed9a-3768-4ee6-b9a4-8dd1953f3f4e.png)

   Kindly follow this [link](https://www.educative.io/answers/how-to-create-an-animated-button-using-html-and-css) for guidance 👍